sample.env

```diff
@@ -83,9 +83,6 @@ MONGO_FROM_EMAIL=support@example.com
 # ODE Wrapper
 WRAPPER_SERVER_PORT=7777
 WRAPPER_CONFIG_ODE_URL=http://ode-url:8080
-WRAPPER_CONFIG_DB_URL=jdbc:postgresql://localhost:5432/dbname?user=username
-WRAPPER_CONFIG_DB_USERNAME=username
-WRAPPER_CONFIG_DB_PASSWORD=<PASSWORD>
 WRAPPER_CONFIG_MAXIMUM_POOL_SIZE=7
 WRAPPER_CONFIG_CONNECTION_TIMEOUT=300000
```

Comment:
Suggestion (non-blocking) (continued): If these aren't used by BasicConfiguration, we should remove them from here also.